administrator/components/com_media/resources/scripts/components/toolbar/toolbar.vue

```diff
@@ -51,7 +51,7 @@
                 return (this.isGridView) ? 'fa fa-list' : 'fa fa-th';
             },
             toggleSelectAllBtnIcon() {
-                return (this.allItemsSelected) ? 'fa fa fa-square-o' : 'fa fa-check-square-o'
+                return (this.allItemsSelected) ? 'fa fa-check-square-o' : 'fa fa fa-square-o'
```

Comment:
why two `fa`

Reply:
I just exchanged ```fa fa fa-square-o``` and ```fa fa-check-square-o``` 
do one ```fa``` need to be removed?

Reply:
@ciar4n